topics/server-session-auth.md

```diff
@@ -90,6 +90,8 @@ The `session` authentication provider exposes its settings via the [SessionAuthe
 ### Step 4: Save user data in a session {id="save-session"}
 
 To save information about a logged-in user to a session, use the [call.sessions.set](server-sessions.md#use_sessions) function.
+> The definition of `auth-form` authentication isn't detailed on this page; it is provided here solely for illustrative purposes. 
```

Comment:
Instead of the first sentence, perhaps we can note that "this example shows a simple authentication flow using a form.". Then from the next sentence is clear where to find more details.